### ISSUE TITLE
`generic.text.color()` has a small range

### ISSUE THREAD

**Issue by Iain-S** [alan-turing-institute/sqlsynthgen#49]
When testing ssg on the `star_080223` schema, I got errors from the `UNIQUE` (though, non-PK) constraint on `advance_decision_type.care_code`. It turns out that there aren't that many colours... 

A couple of options come to mind:

1. We could consider this a good thing since it highlights the fact that we don't handle unique constraints very cleverly (we ignore them) and open an issue to handle them better.
2. We could switch to `generic.text.words(5)` or similar.

**Comment by Iain-S** [comment]
@myyong @mhauru What are your thoughts on the two solutions (above)?

**Comment by mhauru** [comment]
Agree with May on putting in a temporary solution and also marking this as something to develop by creating an issue.

`generic.text.words()` should only generate from a list of "non-bad" words, which can be found here: https://github.com/lk-geimfari/mimesis/blob/master/mimesis/data/en/text.json
It just seems that there are some words in that list that should probably rather be in the "bad" category. (There are also many words in the "bad" category that I don't know why they are there, but that's more amusing than problematic.)

**Comment by mhauru** [comment]
Raised a Mimesis issue: https://github.com/lk-geimfari/mimesis/issues/1320

**Comment by mhauru** [comment]
I submitted a PR for mimesis and it got merged, the NSFW words we ran into when using `generic.text.word()` have now been cleaned up. There might still be others that I didn't pick up.

**Comment by Iain-S** [comment]
Shall we swap to `generic.text.word()` or e.g. `"".join(generic.text.words(5))`?

**Comment by mhauru** [comment]
Yeah, I would risk it. Need to reference a particular commit from github for the mimesis version.
